### PR TITLE
Removed CgroupInfo from isolator reply protobuf.

### PR DESCRIPTION
**Pull request by ConnorDoyle**
As discussed in the design doc.

https://github.com/intelsdi-x/kubernetes/pull/38/files#diff-d1cc24af9cbf835a4cc8670a7e4e67e6L58

### CODE REVIEW

File: pkg/api/v1/generated.proto [hunk 1]
```diff
@@ -899,7 +899,7 @@ message EnvVar {
 // EnvVarSource represents a source for the value of an EnvVar.
 message EnvVarSource {
   // Selects a field of the pod: supports metadata.name, metadata.namespace, metadata.labels, metadata.annotations,
-  // spec.nodeName, spec.serviceAccountName, status.hostIP, status.podIP.
+  // spec.nodeName, spec.serviceAccountName, status.podIP.
```
Comment (flyingcougar) [inline pkg/api/v1/generated.proto]: @ConnorDoyle why there is diff here ?

**Review by flyingcougar — COMMENTED** [review]
lgtm, please check the change in `generated.proto`

**Review by flyingcougar — APPROVED** [review]

**Comment by PiotrProkop** [comment]
LGTM

File: pkg/api/v1/generated.proto (hunk 1 above)
Comment (ConnorDoyle) [reply pkg/api/v1/generated.proto]: Will drop from the diff.

**Pull request merged**